src/main/resources/sql/V10__Index_on_index_table.sql

```diff
@@ -0,0 +1,7 @@
+create index if not exists index_name on index (name);
+create index if not exists index_key on index (key);
+create index if not exists index_sha256hex on index (sha256hex);
+create index if not exists index_start_entry_number on index (start_entry_number);
+create index if not exists index_end_entry_number on index (end_entry_number);
+create index if not exists index_start_index_entry_number on index (start_index_entry_number);
+create index if not exists index_end_index_entry_number on index (end_index_entry_number);
```

Comment:
Are most of the look ups on combined sets of these columns? If so, the index would need to be across multiple columns? I think we need to revisit this. Adding an index to ever individual column could really slow down loads when the database gets large.